Formula/osx-cpu-temp.rb

```diff
@@ -0,0 +1,15 @@
+class OsxCpuTemp < Formula
+  desc "Outputs current CPU temperature for OSX"
+  homepage "https://github.com/lavoiesl/osx-cpu-temp"
+  url "https://github.com/lavoiesl/osx-cpu-temp/archive/1.1.0.tar.gz"
+  sha256 "94b90ce9a1c7a428855453408708a5557bfdb76fa45eef2b8ded4686a1558363"
+
+  def install
+    system "make"
+    bin.install "osx-cpu-temp"
+  end
+
+  test do
+    system "#{bin}/osx-cpu-temp"
```

Comment:
Can we make some kind of check on the output of this command ?

Reply:
Something like this? (Provided the output is like `56.7°C`)
```
assert_match "°C", shell_output("#{bin}/osx-cpu-temp")
```

Reply:
Sounds good, I'll add this. Maybe with a `-C`, to force it to celcius, just in case it follows locale.